docs/source/en/quantization/gguf.md

```diff
@@ -77,3 +77,40 @@ Once installed, set `DIFFUSERS_GGUF_CUDA_KERNELS=true`  to use optimized kernels
 - Q5_K
 - Q6_K
 
+## Using Diffusers checkpoints
+
+You can convert a Diffusers checkpoint to GGUF and use it to perform inference. Use the Space below to
+run conversion:
+
+<script
+	type="module"
+	src="https://gradio.s3-us-west-2.amazonaws.com/5.41.1/gradio.js"
+></script>
+
+<gradio-app src="https://diffusers-internal-dev-diffusers-to-gguf.hf.space"></gradio-app>
+
+Once it is obtained, you can run inference:
+
+```py
+import torch
+
+from diffusers import FluxPipeline, FluxTransformer2DModel, GGUFQuantizationConfig
+
+ckpt_path = (
+    "https://huggingface.co/sayakpaul/different-lora-from-civitai/blob/main/flux_dev_diffusers-q4_0.gguf"
+)
+transformer = FluxTransformer2DModel.from_single_file(
+    ckpt_path,
+    quantization_config=GGUFQuantizationConfig(compute_dtype=torch.bfloat16),
```

Comment:
For Diffusers checkpoints at the moment, we don't auto-fetch the config (don't want to have to track two sets of keys in single file if we can avoid it). So we need to provide a repo id to fetch the config. 

```suggestion
    quantization_config=GGUFQuantizationConfig(compute_dtype=torch.bfloat16),
    config="black-forest-labs/FLUX.1-dev",
    subfolder="transformer",
```